neo/SmartContract/StandardService.cs

```diff
@@ -639,6 +640,23 @@ protected bool Contract_GetStorageContext(ExecutionEngine engine)
             return false;
         }
 
+        private bool Contract_Call(ExecutionEngine engine)
+        {
+            StackItem item0 = engine.CurrentContext.EvaluationStack.Pop();
+            ContractState contract;
+            if (item0 is InteropInterface<ContractState> _interface)
+                contract = _interface;
+            else
+                contract = Snapshot.Contracts.TryGet(new UInt160(item0.GetByteArray()));
+            if (contract is null) return false;
+            ExecutionContext context_new = engine.LoadScript(contract.Script, engine.CurrentContext.ScriptHash, 1);
+            StackItem item1 = engine.CurrentContext.EvaluationStack.Pop();
```

Comment:
We could read this from the manifest

Reply:
We will verify the count of parameters from manifest. (when manifest is ready)

If the number is not equal to 2, an exception will be thrown. This is done to unify the entry points.